core/src/main/java/org/opensearch/sql/executor/pagination/CanPaginateVisitor.java

```diff
@@ -88,19 +97,13 @@ public Boolean visitAggregation(Aggregation node, Object context) {
     return Boolean.FALSE;
   }
 
-  // Queries with ORDER BY clause are not supported
-  @Override
-  public Boolean visitSort(Sort node, Object context) {
-    return Boolean.FALSE;
-  }
-
-  // Queries without FROM clause are not supported
+  // For queries without FROM clause:
   @Override
   public Boolean visitValues(Values node, Object context) {
-    return Boolean.FALSE;
+    return Boolean.TRUE;
   }
 
-  // Queries with LIMIT clause are not supported
+  // Queries with LIMIT/OFFSET clauses are unsupported
```

Comment:
```suggestion
  // Queries with LIMIT and OFFSET clauses are unsupported
```

Using a slash could imply LIMIT or OFFSET, rather than LIMIT and OFFSET, this will make things clearer

Reply:
Good idea, but it has to be applied on the original PR #1599 
My next PR adds support for `LIMIT`, so only `OFFSET` remains unsupported without any ambiguity